chromax/functional.py

```diff
@@ -142,7 +142,7 @@ def select(
     population: Population["n"],
     k: int,
     f_index: Callable[[Population["n"]], Float[Array, "n"]],
-) -> Population["k"]:
+) -> (Population["k"], Int[Array, "k"]):
```

Comment:
Can you change `(Population["k"], Int[Array, "k"])` to be `Tuple[Population["k"], Int[Array, "k"]]`, where Tuple is imported from typing?

Reply:
@younik cool, updated as your suggest. What do you think now?

Reply:
@oddoking looks good, thanks!
There is a small typo + can you also update the typing of `Simulator.select` accordingly?

Reply:
Also tests, should be updated accordingly (basically, every time there is `pop = simulator.select(...)`, we should change to `pop, _ = simulator.select(...)` to unpack args